CRM/Admin/Page/Job.php

```diff
@@ -142,6 +142,10 @@ public function run() {
    * @param null $action
    */
   public function browse($action = NULL) {
+    // check if non-prod mode is enabled.
+    if (Civi::settings()->get('isProductionEnvironment') === FALSE) {
+      CRM_Core_Session::setStatus(ts('Execution of scheduled jobs have been turned off.'), ts("Non-production Environment"), "warning", array('expires' => 0));
```

Comment:
Just to be precise about things: when production environment is toggled off, execution of scheduled jobs is turned off by default, BUT parameters on jobs can override this. So please change message to 'Execution of scheduled jobs has been turned off by default since this is a non-production environment. You can override this for particular jobs by adding runInNonProductionEnvironment=TRUE as a parameter.'